src/pb/ffi.rs

```diff
@@ -3,7 +3,7 @@
 // SPDX-FileContributor: Mathias Kraus
 
 use crate::pb::PublisherOptions;
-use crate::IceOryxError;
+use crate::IceoryxError;
```

Comment:
According to our marketing iceoryx is always written with a non capital letter in the beginning. Don't you dare doodle around with our marketing :smiley_cat: